mastracode/src/agents/workspace.ts

```
@@ -10,6 +10,7 @@ import { Workspace, LocalFilesystem, LocalSandbox, createWorkspaceTools } from '
 import type { LSPConfig, WorkspaceToolsConfig } from '@mastra/core/workspace';
 import { DEFAULT_CONFIG_DIR } from '../constants.js';
 import { loadSettings } from '../onboarding/settings.js';
+import { readHarnessState } from '../utils/harness-state.js';
```

ESLint

[error] `../utils/harness-state.js` import should occur after import of `../tool-names.js`

(import/order)